uber/models.py

```diff
@@ -602,7 +602,7 @@ def checklist_status(self, slug, department):
             }
 
         def jobs_for_signups(self):
-            fields = ['name', 'location_label', 'description', 'weight', 'start_time_local', 'duration', 'weighted_hours', 'restricted', 'extra15', 'taken']
+            fields = ['name', 'location_label', 'description', 'weight', 'start_time_local', 'end_time_local', 'duration', 'weighted_hours', 'restricted', 'extra15', 'taken']
```

Comment:
This fixes the shift end times not being correct -- the calendar didn't know when shifts ended because we never put that in the dictionary it uses!